after_success.sh

```diff
@@ -19,8 +19,34 @@ else
         start_section "package.upload" "${GREEN}Package uploading...${NC}"
         # Test `anaconda` with ANACONDA_TOKEN before uploading
         source $GITHUB_WORKSPACE/.github/scripts/test_anaconda.sh
+
+        os_package_match='conda-bld/(.*)'
+        [[ $CONDA_OUT =~ $os_package_match ]]
+        os_and_package="${BASH_REMATCH[1]}"
+
+        name_version_match='conda-bld/(.*)/(.*)-(.*)-'
```

Comment:
Did you check this with a package having hyphenated name like `nextpnr-xilinx`?

I'm wondering if the expression should be more precise about whether each part can contain a hyphen, like maybe `conda-bld/(.*)/(.*)-([^-]*)-[^-]*$`? Hyphen is forbidden in version and build so those will always be after the last two hyphens.

Reply:
Updated the regex expression, following your suggestion.
Checked with capnproto-java, works like a charm:
https://github.com/litex-hub/litex-conda-misc/runs/1702341950?check_suite_focus=true